mne/viz/misc.py

```diff
@@ -445,14 +464,14 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
 
     Parameters
     ----------
-    subject : str
+    subject : str | None
```

Comment:
This very function here, `plot_bem()`, is not a method of a class. I've tried to make use of a the docdict anyway, LMK what you think. Changes in 33f8ab8

Reply:
Isn't `subject=None` not supported, then?

Reply:
Good catch, I checked the function(s) it calls and they expect a string, not `None`. I had simply adjusted the docstring here because the default param value in the function signature was `subject=None`. So this is actually a bug. I fixed this in bba68f6